### PR TITLE
Respace Species Code

### PR DESCRIPTION
**Pull request by VMSolidus**
_"Edited from its original purpose"_

### Summary
This PR was originally going to be a complete removal of any mention of DeltaV/Nyanotrasen/Simplestation from the codebase, and instead I have elected to significantly reduce the scope of this PR. Instead of blanket doing all rebrandings in a single PR, this PR simply acts to move all Species-specific code from their original locations, to a single unified location that is identical across Server, Shared, and Client. This takes the form of the newly repurposed Species folder. The directory should be set to be the following:

Content.X.Species
>One folder for each playable species, eg: Human, Felinids, Harpy, etc.
>>Components
>>Systems
>>Cvars 

This is not going to represent the final location for each of the Species specific code. But it is going to serve as a suitable location to store all currently existing species-specific systems. I will reorganize other parts of the repository in separate PRs

### CODE REVIEW

**Comment by VMSolidus** [comment]
I have updated both the title, summary, and objectives of this PR to reflect a more realistic scope.

**Comment by VMSolidus** [comment]
image
Wew, and now I have tests plus compiler pass. This is nearly as far as I can tell, all of the species code that previously existed spread across DeltaV, Nyanotrasen, and Simplestation14, now placed inside Content.X.Species. Not all of it should stay there indefinitely, but for the sake of organization they're all now in a unified place that's easy to find.

File: Content.Client/Species/Felinids/Systems/PseudoItemSystem.cs
Comment (DEATHB4DEFEAT) [inline Content.Client/Species/Felinids/Systems/PseudoItemSystem.cs:1]: Rename the `Felinids` folder to `Felinid`

File: Content.Server/Species/Felinids/Components/CoughingUpHairballComponent.cs
Comment (DEATHB4DEFEAT) [inline Content.Server/Species/Felinids/Components/CoughingUpHairballComponent.cs:1]: Rename the `Felinids` folder to `Felinid`

File: Content.Server/Species/Spider/Systems/SpiderSystem.cs
Comment (DEATHB4DEFEAT) [inline Content.Server/Species/Spider/Systems/SpiderSystem.cs:1]: I think this is used by things other than Arachne, put it back

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

**Comment by VMSolidus** [comment]
I'm closing this as per a notice for our new policy regarding code respacing. As stated on our discord at this link:
https://discord.com/channels/1218698320155906090/1218698321053356059/1224795186689867818

**Pull request closed without merging**